client/state/sites/plans/reducer.js

```diff
@@ -34,6 +35,9 @@ export function plans( state = {}, action ) {
 			} );
 		case REMOVE_SITE_PLANS:
 			return omit( state, action.siteId );
+		case SERIALIZE:
+			//TODO: we have full instances of moment.js on sites.plans[siteID].data
```

Comment:
@scruffian, @stephanethomas, @drewblaisdell - it looks like we need to refactor a bit code related to plans and purchases to apply `moment.js` outside of stores.